### ISSUE TITLE
[homematic] ClassCastException when parsing XMLResponse from Lightify-Devices in HM

### ISSUE THREAD

**Issue by KaaNee** [openhab/openhab-addons#5164]
OsramLightify-Devices binded to the homematic-base will also discovered by homematic-binding. When adding them as a thing, it will be added but kept in Offline (java.lang.String cannot be cast to java.util.Map) State and were not usable.

Technical:
While adding them as things, an Exception is thrown as an XMLResponse contains not the normal expected Array/Map of Information when querying the Lightify-Devices. As this is not checked in GetParamsetParser an ClassCastException is thrown and the connecting process is aborted.
When checking if first element (which is used only) of message-Array is type of Map (or not) everything went fine and devices are online.
In this case, the element of the message-Array is an empty String, not a Map.

tldr; just checked type before using it.

I'll create a PR afterwards.

Regards,
Kai

[stacktrace_cce_homematic_lightify.log](https://github.com/openhab/openhab2-addons/files/2987374/stacktrace_cce_homematic_lightify.log)

**Comment by MHerbst** [comment]
The problem as been solved with PR #5165 and can be closed, ping @openhab/add-ons-maintainers

**Comment by hmerk** [comment]
closed as was solved with PR #5165
